eslint/eslint-plugin-security/src/no-unsafe-regexp.test.ts

```diff
@@ -6,7 +6,8 @@ import { noUnsafeRegExp } from './no-unsafe-regexp';
 
 const { RuleTester } = ESLintUtils;
 const ruleTester = new RuleTester({
-  parser: '@typescript-eslint/parser'
+  // In PNPN 7.0 using '@typescript-eslint/parser' unexpectedly fails, se we are resolving the absolute path of the parser
```

Comment:
```suggestion
  // In PNPM 7.0 using '@typescript-eslint/parser' unexpectedly fails, so we are resolving the absolute path of the parser
```

Reply:
Same with the way it is used in eslint-plugin

Reply:
Also, could you provide some context in the comment as to why it fails?

Reply:
Switched to a more elaborate comment